docs/projects/sleep-sort/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Sleep Sort in Every Language
+layout: default
+date: 2019-10-08
+last-modified: 2019-10-08
+featured-image:
+tags: [sleep-sort]
+authors:
+  - agilob
```

Comment:
Whoops! One second. Are you in the author file? https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml

Reply:
Updated